### ISSUE TITLE
Add a split_tld function

### ISSUE THREAD

**Issue by Yomguithereal** [barseghyanartur/tld#42]
This is a common use case.

```python
domain, tld = split_tld(url)
```

But what if tld does not exist?

**Comment by barseghyanartur** [comment]
What about

```python

from tld import get_tld

try:
    obj = get_tld(url, as_object=True)
    domain = obj.domain
    tld = obj.tld
except Exception:
    domain = None
    tld = None
```

It can be simply wrapped into a helper function, which would return ``None, None``.

**Comment by barseghyanartur** [comment]
To be implemented in 0.9.

**Comment by barseghyanartur** [comment]
Released in 0.9.

**Comment by barseghyanartur** [comment]
How it works:

```python
from tld import parse_tld

tld, domain, subdomain = parse_tld('http://www.google.com')
# 'com', 'google', 'www'
```
